vendir.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+directories:
+  - path: vendor
+    contents:
+      - path: cilium
+        git:
+          url: https://github.com/giantswarm/cilium
+          ref: main
+        includePaths:
+          - install/kubernetes/cilium/**/*
+  - path: helm/cilium
+    contents:
+      - path: .
+        directory:
+          path: vendor/cilium/install/kubernetes/cilium
```

Comment:
As I mentioned in my other comment about the `rm` in the Makefile, this is not really needed since you are copying the entire directory. In other repos, we were doing this because we only copied the templates and some other things but not the main chart directory)